src/Components/SigDetailsTable/SigDetailsTable.js

```diff
@@ -32,6 +32,7 @@ import MessageState from '../MessageState/MessageState';
 import messages from '../../Messages';
 import EmptyAccount from '../SharedComponents/EmptyAccount';
 import './SigDetailsTable.scss';
+import { useWorkspaceFeatureFlag } from '../../Utilities/Hooks';
 
```

Comment:
In this file lives a Group Filter, around line 151, this needs to be updated 

![Screenshot 2024-06-24 at 3 30 06 PM](https://github.com/RedHatInsights/malware-detection-frontend/assets/60629070/954e22dc-f8a5-4124-90ac-21edab02dc08)